lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9094,6 +9096,47 @@ class SwiftifyInfoPrinter {
 };
 } // namespace
 
+namespace {
+  /// Look for any side effects within a Stmt.
```

Comment:
Oops, I just realized that I forgot to remove this stray comment that got copied over when I was setting up the `StmtVisitor` boilerplate 😮‍💨